packages/sdks/src/helpers/subscribe-to-editor.ts

```diff
@@ -38,6 +40,15 @@ export const createEditorListener = ({
           callbacks.animation(data.data);
           break;
         }
+        case 'builder.resetState': {
+          const messageContent = data.data;
+          const modelName = messageContent.model;
+          const newState = messageContent?.state;
+          if (modelName === model && newState) {
```

Comment:
Added extra check around `modelName` to match other callbacks.

FYI, @samijaber @sidmohanty11